.travis.yml

```diff
@@ -7,8 +7,20 @@ env:
   - SKIP_INTEGRATION=true GO111MODULE=on GOPROXY=https://proxy.golang.org GOSUMDB=sum.golang.org
 
 script:
-  - ./hack/check-go-sum.sh
+  - ./hack/ci.sh
```

Comment:
what can we remove from this file if ./hack/ci.sh is called? Ideally all the rest of it?

Reply:
Good catch, I meant to have this be ci.sh only